crates/client/rpc/src/constants.rs

```diff
@@ -2,3 +2,16 @@
 pub const MAX_EVENTS_KEYS: usize = 100;
 /// Maximum number of events that can be fetched in a single chunk for the `get_events` RPC.
 pub const MAX_EVENTS_CHUNK_SIZE: usize = 1000;
+
+/// Class hashes whose corresponding contracts classes are "NoValidateAccount".
+pub const NO_VALIDATE_ACCOUNT_CLASS_HASHES: [&str; 2] = [
+    "0x0279d77db761fba82e0054125a6fdb5f6baa6286fa3fb73450cc44d193c2d37f",
+    "0x35ccefcf9d5656da623468e27e682271cd327af196785df99e7fee1436b6276",
+];
+/// Predeployed accounts adresses defined in the genesis state.
+pub const PREDEPLOYED_ACCOUNTS_ADDRESSES: [&str; 4] = ["0x1", "0x2", "0x3", "0x4"];
+
+/// Pk for the relevant predeployed accounts defined in the genesis state.
+pub const ARGENT_PK: &str = "0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d";
+/// Fee token contract address defined in the genesis state.
+pub const FEE_TOKEN_ADDRESS: &str = "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7";
```

Comment:
Alright, but idk where such a function should be implemented.
`print_development_accounts` only has access to a GenesisLoader (which itself needs a base path to find the genesis assets), while `Starknet<...>` in the file `client/rpc/src/lib` doesn't have access to a path towards `genesis.json`, but can use  RPCs to query the state.

Reply:
Hmm, you're right. Actually, the idea of reading the pre funded accounts as specific indexes in the genesis state doesn't make sense. If someone forks Madara and changes the genesis state, this would break. Let's move these variables to constants and add a utils function to load them. We can add the utils function to the client itself or create a utils directory on the root level. I think the latter existed before and was removed for some reason. @tdelabro what do you think?

Reply:
This whole thing is super hacky anyway.
I think the easiest thing we can do is to return the GenesisConfig struct, without the whole nice printing around it

Reply:
Also, is this thing required? 
The printing of the deployed account when running in dev mode is a cool nice to have for developers trying to add TX to the network. But what is the usecase of having an RPC to do so? Who asked for it?

Reply:
My point is that if nobody needs it and the implementation has to be super hacky, don't do it.

Reply:
Braavos needs it. They call this endpoint to get the available accounts and accordingly use it for faucets. Katana and devnet already support this #1230

Reply:
> I think the easiest thing we can do is to return the GenesisConfig struct, without the whole nice printing around it

You mean return the GenesisConfig from the pallet to the client when an RPC call is made and remove the printing on start?

Reply:
We can easily have the list of deployed accounts by iterating over the storage.
If we do this at block 0 we have the predeployed accounts.
But we won't get information about "this one is argent, this one is Cairo0NoValidate". Is this fine?

Reply:
The whole point of the genesisloader was to not have this in a hardcoded way.

Reply:
If we embrace the genesis loader, we should do it entirely.
We can complexity the struct by adding for which contract it contains a `name` and optional `private_key` field.
This way we can read it from disk, return it and anybody can display or use it the way he want

Reply:
Fair, but I don't think we should add extra fields to the contracts section. Can we add a new field to `GenesisData` called `prefunded_accounts` where the user can explicitly state the `address` and the `pk`.

Reply:
```
pub struct GenesisData {
    pub contract_classes: Vec<(ClassHash, ContractClass)>,
    pub contracts: Vec<(ContractAddress, ClassHash)>,
    pub storage: Vec<(ContractStorageKey, StorageValue)>,
    pub fee_token_address: ContractAddress,
    pub seq_addr_updated: bool,
}
```

Why couldn't we add them to the `contracts` tuple?

Reply:
`    pub contracts: Vec<(ContractAddress, ClassHash, Option<String>, Option<Vec<u8>>)>,`

Reply:
I think it tightly couples the `contracts` information with prefunded accounts. How do I add a pre funded account which I don't want to be displayed to the user? Also, if I have very little prefunded accounts (as many chains will have in production), it doesn't make sense for me to repeatedly write `false` or `vec![]`

Reply:
@Juul-Mc-Goa let's add `prefunded_accounts` to the `GenesisData` in that case. We can return the genesis information from the pallet for the client to use. Will you be able to pick this?

Reply:
> We can return the genesis information from the pallet for the client to use.

I'm not sure about this. I think the file has to be read again.

Reply:
> @Juul-Mc-Goa let's add `prefunded_accounts` to the `GenesisData` in that case. We can return the genesis information from the pallet for the client to use. Will you be able to pick this?

I'm not sure how to do this: 
1. it seems that modifying `GenesisData` implies modifying `genesis.json` (because this file is deserialized into a GenesisData object),
2. `GenesisData` itself is only used to get a `GenesisConfig` object, so that would imply modifying the definition of `GenesisConfig`,
3. `GenesisConfig` is used only through the `build()` method, which writes stuff to the pallet storage. So I assume there should be another `pallet::storage` for predeployed accounts in `pallets/starknet/src/lib.rs` ? Otherwise, maybe we can use a `pallet::constant` in the definition of `Config`, but idk how one should write constants to the config when building GenesisConfig.

Reply:
I'm fine with it. The genesis.json fine is copied under the chain data directory, so it should stay there an not be touched or removed during the whole time the chain run